ThreeBotPackages/vesting_service/actors/vesting_service.py

```diff
@@ -70,6 +70,15 @@ def _is_multisig_account(self, address: str) -> bool:
         resp = horizon_server.accounts().account_id(address).call()
         return len(resp["signers"]) != 1
 
+    def _check_has_vesting_account(self, address: str):
+        accounts = get_wallet()._get_horizon_server().accounts()
+        accounts_for_signer = accounts.for_signer(address).call()
+        for record in accounts_for_signer["_embedded"]["records"]:
+            if "tft-vesting" in record.get("data"):
+                decoded_data = j.data.serializers.base64.decode(record["data"]["tft-vesting"]).decode()
+                if decoded_data == "here comes the formula or reference":  # TODO should it be this sentence?
```

Comment:
VESTING_SCHEME

Reply:
and a check if the rest of the signers is correct, if not I can trick you in to sending to an account where I have signing rights on and steal your tft

Reply:
@robvanmieghem  I added the check in https://github.com/threefoldfoundation/tft-stellar/pull/336/commits/053ad40dcbc8eccee7806b9acb3d63453f27b656